kernel_gateway/services/activity/manager.py

```diff
@@ -48,35 +75,76 @@ def get_map_for_kernel(self, kernel_id):
         return self.values[kernel_id]
 
     def publish(self, kernel_id, activity_type, value=None):
-        '''Sets the value stored for *activity_type*. If the activity_type is not found, it
-        will be created and assigned to *value*.
-        '''
-        self.get_map_for_kernel(kernel_id)[activity_type] = value;
+        """Sets the `value` stored for `activity_type` for `kernel_id`.
 
+        Parameters
+        ----------
+        kernel_id : str
+            Unique identifier for the kernel
+        activity_type : str
+            Activity key to set
+        value : any
+            Value to set for the activity
+        """
+        self.get_map_for_kernel(kernel_id)[activity_type] = value
 
     def increment_activity(self, kernel_id, activity_type):
-        '''Increments the value stored for *activity_type*. If the value currently stored
-        is not an int, an TypeError will be raised. If the activity_type is not found, a
-        KeyError will be raised.
-        '''
-        self.get_map_for_kernel(kernel_id)[activity_type] += 1;
+        """Increments the int value stored for `activity_type` for `kernel_id`.
```

Comment:
Increments the `activity_type` value, where value is an int type, for `kernel_id`.